src/utils/utils.ts

```diff
@@ -26,3 +26,23 @@ export function parseName(nameStr: string) {
 export function pad(v: number) {
   return v.toString().padStart(2, '0');
 }
+
+export function shuffle<T>(array: T[]): T[] {
```

Comment:
```suggestion
export function shuffle<T>(originalArray: T[]): T[] {
    const array = originalArray.slice();
```
sort 야 원래 생긴게 그런 애니까 별 수 없지만 기왕이면 return 을 위한 array 를 따로 만들어서 반환하면 좋을 것 같습니다... 사실 이런 프로그램에서는 솔직히 아무 상관 없기는 하지만요...